_posts/2020-04-28-may-meetup.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "May 2020 - Remote again! 💪🏼"
+date:   2020-04-29 00:00:00
+categories: events
+---
+
+**When?**
+
+Thurday, 07.05.2020, 19:00
+
+**Where?**
+
+REMOTE!
+
+As all of you know, we’re stuck at home due to Corona Virus 🦠.
+
+About that, we hope you’re all safe and healthy, as well as you family and friends!
+
+To keep you all distracted and to be able to talk about our favorite nerd topics, we’re not going to give up on our monthly meetup! 💪🏼
+
+As our last months online event worked very well 🎉 we are going to meet remotely again. We are currently evaluating different video call tools to see which one to use and will update here in time.
+
+Also, advices and tipps from you are very welcome! 🙏🏼
+
+This months event is all about Combine!
+
+**Schedule**
+
+19:00 - Virtual "doors" open
+
+Let's gather in Slack!
+
+Click here 👉🏼 https://slack.cocoaheads.hamburg
+to join our workspace 'cocoaheadshamburg.slack.com' to enable an easier communication.
+
+19:30 - Meeting starts 🎉
+
+Everyone joins the video call.
+
+19:45 - 1️⃣ Talk: Learning Combine by refactoring a networking layer - Donny Walls
+In this talk Donny will explain the basics of Combine, and how they can be applied to refactor an existing networking layer into one that uses Combine. Attendees will learn about Combine’s Publishers, Subscribers, operators, the Cancellable protocol and more.
```

Comment:
You'll need an empty line before this if you want the line-break in the result.

Same for the 2nd talk description